### DISCUSSION TITLE
How to send our notebook to this repo ?

### DISCUSSION THREAD

**Discussion by Marioherreroglez** [obidam/ds2-2022#7] [Projects]
Hello, what is the best way to push/pull our branches?

**Comment by gmaze** [comment]
Once you're ready to share your project notebook, you have to push it to the class repo ds2-2022

These are the steps to do it:

### Get a synchronized version of the repo on your account to work with:
- [ ] Fork this github repository to your own account (log in, go to [ds2-2022](https://github.com/obidam/ds2-2022), click on the fork button in the upper right)
- [ ] you're now redirected to your copy of the class repo like: ``https://github.com/<YOUR_GITHUB_LOGIN>/ds2-2022``

### Work on your project and upload results
Once you're happy with your notebook, this is the way to put online with the web interface:
- [ ] navigate to the ``ds2-2022/projects`` folder
- [ ] select the ``upload files`` from the ``Add files`` button in the upper right of the page, 
- [ ] drag and drop your notebook file
- [ ] select `` Commit directly to the main branch`` and click on the green button to Commit your changes

### Send your results back to the class repo
- [ ] Get back to your fork at ``https://github.com/<YOUR_GITHUB_LOGIN>/ds2-2022``
- [ ] and go to the ``Pull Requests`` tab: ``https://github.com/<YOUR_GITHUB_LOGIN>/ds2-2022/pulls``
- [ ] Click on the ``New pull request`` button and then on ``Create pull request``
- [ ] Set a title to your pull request (use your last names and project number, eg: *Project 33 - Maze & Dow*) and click on the ``Create pull request`` button to finish the process
- [ ] your pull request (PR) should then be visible on the class repo page: https://github.com/obidam/ds2-2022/pulls
- [ ] wait for @gmaze to merge your PR, or not if more information are necessary

**Comment by gmaze** [comment]
# Further Note
If I/we ask you some modifications in the PR page discussion, just get back to your fork on your account to make these updates.
They will reflect automatically in you commits, **you don't have to create a new pull request !**
